layers/+tools/transmission/README.org

```diff
@@ -63,52 +64,68 @@ Add =(transmission :variables transmission-auto-refresh-all t)= to
 
 ** =transmission-mode=
 
-| Key binding | Description                                                   |
-|-------------+---------------------------------------------------------------|
-| ~SPC m g t~ | Open a `transmission-files-mode' buffer for torrent at point. |
-| ~SPC m a t~ | Add torrent by filename, URL, magnet link, or info hash.      |
-| ~SPC m s d~ | Set global download speed limit in kB/s.                      |
-| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
-| ~SPC m g i~ | Open a =transmission-info-mode= buffer for torrent at point.  |
-| ~SPC m s l~ | Set global seed ratio limit.                                  |
-| ~SPC m m~   | Move torrent at point or in region to a new location.         |
-| ~SPC m r~   | Prompt to remove torrent at point or torrents in region.      |
-| ~SPC m t s~ | Toggle torrent between started and stopped.                   |
-| ~SPC m a a~ | Add announce URLs to torrent or torrents.                     |
-| ~SPC m s u~ | Set global upload speed limit in kB/s.                        |
-| ~SPC m v~   | Verify torrent at point or in region.                         |
-| ~SPC m q~   | Quit and bury the buffer.                                     |
-| ~SPC m s b~ | Set bandwidth priority of torrent(s) at point or in region.   |
+| Key binding | Description                                                       |
+|-------------+-------------------------------------------------------------------|
+| ~H~         | Move torrent to the top of the queue.                             |
```

Comment:
You can group J/K and H/L so it's clearer

Reply:
This is an easy task, I'll get this done up!

Reply:
This has been settled!